### PR TITLE
objdump: add intel syntax command

### LINKED ISSUE #8389: objdump: intel syntax
Use -M intel flag

### PR DESCRIPTION
**Pull request by jxu**
Also remove hyphen from disassemble to use more common form.

Maybe make a mention intel syntax applies to x86 asm but that should be obvious.

Resolves #8389

### CODE REVIEW

File: pages/common/objdump.md [hunk 1]
```diff
@@ -7,10 +7,14 @@
 
 `objdump -f {{binary}}`
 
-- Display the dis-assembled output of executable sections:
+- Display the disassembled output of executable sections:
 
 `objdump -d {{binary}}`
 
+- Disaply the disassembled executable sections in intel syntax:
```
Comment (kbdharun) [inline pages/common/objdump.md:14]:
```suggestion
- Display the disassembled executable sections in intel syntax:
```

Comment (jxu) [reply pages/common/objdump.md]: Oops! that's embarrassing.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks @jxu . Let's see what others think.

**Review by mfrw — APPROVED** [review]
Although, we prefer long options, but given this page uses short options; LGTM :)
Thank you for your contrinution @jxu

File: pages/common/objdump.md [hunk 2]
```diff
@@ -7,10 +7,14 @@
...
 `objdump -f {{binary}}`
 
-- Display the dis-assembled output of executable sections:
+- Display the disassembled output of executable sections:
 
 `objdump -d {{binary}}`
 
+- Display the disassembled executable sections in intel syntax:
+
+`objdump -M intel -d {{binary}}`
```
Comment (CleanMachine1) [inline pages/common/objdump.md:16]:
```suggestion
`objdump -M {{intel}} -d {{binary}}`
```

Is there multiple options that can go with -M such as amd

Comment (jxu) [reply pages/common/objdump.md:16]: For x86 there is just Intel or AT&T (default for this and GNU tools). AMD processors are also x86.

Comment (CleanMachine1) [reply pages/common/objdump.md:16]: We should the {{ }} around it since it takes arm I assume.

Comment (jxu) [reply pages/common/objdump.md:16]: ARM doesn't have two versions of syntax like x86. In my original post I made a note that maybe this should be mentioned this only applies for x86 binaries

**Comment by CleanMachine1** [comment]
Thanks for your contribution.

**Pull request merged**